### PR TITLE
added instructions for using teams

### PR DESCRIPTION
**Pull request by nicain**
Add information about subteams

### CODE REVIEW

File: HCK04_2018_Seattle/Projects/README.md [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
+
+- Join the weekly [preparation call](../README.md#preparatory-videoconferences) to discuss your idea.
+- Post questions about project ideas and teams on the [Hackathon forum][forum].
+
+## Designing your team and project:
+
+To facilitate the discussions around project ideas, you can ask the [Hackathon forum maintainers][maintainers]  to create a sub-team for you.  The subteams that already exist are listed [here][subteams].
+- Use the Discussion forum to discuss project scope and team membership.
+- As you work, distill your ideas into a [pinned discussion][pinned-discussion].  If you use the [project description template][project-description-template], you'll be all set up for the next step (below).
+- Ultimately, this space is meant to serve your team, not become a burdien, so you should use it however you want.
```
Comment (jcfr) [inline HCK04_2018_Seattle/Projects/README.md]: burdien -> burden

File: HCK04_2018_Seattle/Projects/README.md [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
+
+## Once your project is finalized:
```
Comment (jcfr) [inline HCK04_2018_Seattle/Projects/README.md]: Once your project is finalized -> Once your project idea is finalized:

File: HCK04_2018_Seattle/Projects/README.md [hunk 3]
```diff
@@ -1,8 +1,20 @@
...
+
+- When your project and team members and are ready, add a new entry in the list of **Projects** by creating a new `README.md` file in subfolder in `Projects` folder, and copying contents of [project description template][project-description-template] file into it. Step-by-step instructions for this are:
```
Comment (jcfr) [inline HCK04_2018_Seattle/Projects/README.md]: When your project and team members and are ready .. 
-> When your project and team members are ready ...  (remove extra `and``)

File: HCK04_2018_Seattle/Projects/README.md [hunk 4]
```diff
@@ -1,8 +1,20 @@
...
-- When you are ready, add a new entry in the list of **Projects** by creating a new `README.md` file in subfolder in `Projects` folder, and copying contents of [project description template][project-description-template] file into it. Step-by-step instructions for this are:
+## First steps:
+
+- Join the weekly [preparation call](../README.md#preparatory-videoconferences) to discuss your idea.
+- Post questions about project ideas and teams on the [Hackathon forum][forum].
+
+## Designing your team and project:
+
+To facilitate the discussions around project ideas, you can ask the [Hackathon forum maintainers][maintainers]  to create a sub-team for you.  The subteams that already exist are listed [here][subteams].
+- Use the Discussion forum to discuss project scope and team membership.
```
Comment (jcfr) [inline HCK04_2018_Seattle/Projects/README.md]: Use the main [Hackathon forum][forum] to discuss ...

**Comment by nicain** [comment]
@jcfr OK, all changes made.  Ill wait on you to merge or request more changes.

File: HCK04_2018_Seattle/Projects/README.md [hunk 5]
```diff
@@ -1,8 +1,20 @@
...
-- When you are ready, add a new entry in the list of **Projects** by creating a new `README.md` file in subfolder in `Projects` folder, and copying contents of [project description template][project-description-template] file into it. Step-by-step instructions for this are:
+## First steps:
+
+- Join the weekly [preparation call](../README.md#preparatory-videoconferences) to discuss your idea.
+- Post questions about project ideas and teams on the [Hackathon forum][forum].
+
+## Designing your team and project:
+
+To facilitate the discussions around project ideas, you can ask the [Hackathon forum maintainers][maintainers]  to create a sub-team for you.  The subteams that already exist are listed [here][subteams].
+- Use the main [Hackathon forum][forum] to discuss project scope and team membership.
```
Comment (jcfr) [inline HCK04_2018_Seattle/Projects/README.md]: I updated my initial comment with `Use your sub-team forum ...` which would be more coherent considering you suggest to create such a sub-team forum above

File: HCK04_2018_Seattle/Projects/README.md [hunk 6]
```diff
@@ -1,8 +1,20 @@
...
+- As you work, distill your ideas into a [pinned discussion][pinned-discussion].  If you use the [project description template][project-description-template], you'll be all set up for the next step (below).
+- Ultimately, this space is meant to serve your team, not become a burden, so you should use it however you want.
```
Comment (jcfr) [inline HCK04_2018_Seattle/Projects/README.md]: @nicain  I think this one should be good one:
```
- Use the main [Hackathon forum][forum] (or your sub-team forum) to discuss project scope and team membership.
```

what do you think ?

Comment (nicain) [reply HCK04_2018_Seattle/Projects/README.md]: Ill make the change.

Comment (nicain) [reply HCK04_2018_Seattle/Projects/README.md]: OK, sub-team change made. Ill wait on you to merge or request more changes.

**Pull request merged**